src/main/java/dk/kb/datahandler/facade/DsDatahandlerFacade.java

```diff
@@ -49,9 +54,80 @@
 
 public class DsDatahandlerFacade {
     private static final Logger log = LoggerFactory.getLogger(DsDatahandlerFacade.class);
-    
+
     private static DsStorageClient storageClient;
-    
+
+    /**
+     * <p>
+     * Will load all referenceId defined in storage and call Kaltura to map them to KalturaId.
+     * The results will be saved in ds-storage in the mapping table.
+     * When all mappings has been updated, the records will enriched with the kalturaId.
+     * If the job fails alll mappings are loaded the records will not be enriched.
```

Comment:
Is this sentence missing an 'and'? ... and the records will not...

Reply:
rewritten